frameworks/hdfs/docs/kerberos.md

```diff
@@ -32,7 +32,7 @@ hdfs/data-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
 HTTP/data-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
 hdfs/data-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
 HTTP/data-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
-HTTP/hdfs.marathon.autoip.dcos.thisdcos.directory@LOCAL
```

Comment:
Should we add a link as to how the `hdfs` part changes if we use the non-default name?

Reply:
Perhaps this is addressed in https://github.com/mesosphere/dcos-commons/pull/1960